core/src/main/java/com/graphhopper/storage/index/LocationIndex.java

```diff
@@ -42,12 +42,6 @@
     LocationIndex prepareIndex();
 
     /**
-     * @return the closest node id for the specified geo location (latitude,longitude)
-     * @deprecated will be removed with 0.8 use 'QueryResult findClosest' instead
```

Comment:
BTW: Why haven't we marked the method with @Deprecated. Maybe someone continued to use them without reading the comment?